### ISSUE TITLE
HWES estimator produces forecasts of much larger magnitude with statsforecast 1.5.0

### ISSUE THREAD

**Issue by fdrhoward** [mikekeith52/scalecast#74]
First time submitting a bug so please excuse...

Using Release 0.18.6

Noticed a friend and I were getting drastically different HWES forecasts (his were orders of magnitude off and clearly appeared to be a bug).  Identified I was running on statsforecast 1.4.0 while he had statsforecast 1.5.0. When I upgraded to 1.5.0 I too received the same bug.

I have uploaded the monthly data being forecasted for reproducability.

```
test_model = Forecaster(
            y = fcst_data['quantity_lbs'], # required
            current_dates = fcst_data['bwg_month_beg'], # required
            future_dates = 12, # length of the forecast horizon
            test_length = .15, # set a test set length or fraction to validate all models if desired
            cis = False, # choose whether or not to evaluate confidence intervals for all models
            metrics = ['rmse','mae','mape','r2'], # the metrics to evaluate when testing/tuning␣ models
    )
    

#Sets Estimators
test_model.set_estimator('hwes')

test_model.manual_forecast(
    trend=None,
    seasonal='mul',
    use_boxcox=True
)


test_model_results = test_model.export(['lvl_fcsts','model_summaries']) 

display(test_model_results['model_summaries'])
display(test_model_results['lvl_fcsts'])
```

[scalecast_bug_data.csv](https://github.com/mikekeith52/scalecast/files/12360973/scalecast_bug_data.csv)

**Comment by mikekeith52** [comment]
Looking into it.

**Comment by mikekeith52** [comment]
I found a similar issue in the [statsmodels GitHub](https://github.com/statsmodels/statsmodels/issues/8680). This might be related to that. From what I can tell, 0.15.0 is not a stable version of statsmodels as of today. So maybe a requirement to use `statsmodels<=0.14.0` is the fix? Any reason why you have to use 0.15.0?
